examples/plotting/AIA_HMI_composite.py

```diff
@@ -0,0 +1,50 @@
+"""
+===================
+Overlaying Two Maps
+===================
+
+This example demonstrates how to overlay two maps to compare features.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.data.sample
+from sunpy.map import Map
+
+###############################################################################
+# We start with the sample data. HMI shows the line-of-sight magnetic field at
+# the photosphere while AIA 171 images show the resulting magnetic fields
+# filled with hot plasma above, in the corona. We want to see what coronal
+# features overlap with regions of strong line-of-sight magnetic fields.
+aia_map = Map(sunpy.data.sample.AIA_171_IMAGE)
```

Comment:
```suggestion
# features overlap with regions of strong line-of-sight magnetic fields.

aia_map = Map(sunpy.data.sample.AIA_171_IMAGE)
```

Will need to check with others but I think that we want a line break here. 
I do wonder if we have a formal guide on style for examples somewhere.

Reply:
Yeah, there should be a line breaks here. It is not mentioned in the [Example Gallery guidelines](https://docs.sunpy.org/en/latest/dev_guide/contents/example_gallery.html), but it is shown in the example template.